routes.py

```diff
@@ -0,0 +1,217 @@
+from flask import Blueprint, request
+import ConfigParser
+from werkzeug import exceptions
+from webargs import fields
+from webargs.flaskparser import FlaskParser
+
+from protorpc import message_types
+from protorpc import messages
+from protorpc import protojson
+
+import authentication
+from bond import Bond
+from fence_token_vending import FenceTokenVendingMachine
+from fence_api import FenceApi
+from open_id_config import OpenIdConfig
+from sam_api import SamApi
+from oauth_adapter import OauthAdapter
+from status import Status
+import json
+import ast
+
+
+class Parser(FlaskParser):
+    DEFAULT_VALIDATION_STATUS = 400
+
+
+parser = Parser()
+use_args = parser.use_args
+
+
+class JsonField(messages.StringField):
```

Comment:
Probably outside the scope of this ticket (since it's existing code), but with all these classes here, I wonder if it would be helpful to make it clear which are public/protected/private attributes so that it's a bit clearer how to / not to use them. Then again, routes is only imported by main.py, so ¯\_(ツ)_/¯

Reply:
Yeah, I was thinking that it'd be nice to pull all of them out of `routes.py` and in to a different file (`responses.py` or something like that) and then import them to `routes.py`, but I think it is probably outside of the scope of this ticket since this was essentially just copied over from `main.py`